types/include/interpolation.h

```diff
@@ -23,6 +23,8 @@ rage_FrameNo rage_interpolated_view_get_pos(
 void rage_interpolated_view_advance(
     rage_InterpolatedView * view, rage_FrameNo n_frames);
 
+#define RAGE_VALIDITY_ADD(a, b) (a == UINT32_MAX) ? a : (b == UINT32_MAX) ? b : a + b
```

Comment:
Is this a sneaky way to create a polymorphic inline function?

Reply:
Strictly speaking, no, because you can pass arbitary lumps of code as the arguments.
But in this context that's kinda what I'm doing (although I don't (and in this case musn't) use the polymorphism).